README_zh-CN.md

```diff
@@ -112,6 +114,7 @@ MMDetection3D 是一个基于 PyTorch 的目标检测开源工具箱, 下一代
 | Group-Free-3D        | ✗        | ✗        | ✗        | ✓         | ✗     | ✗        | ✗     |
 | ImVoxelNet           | ✓         | ✗        | ✗        | ✗        | ✗     | ✗        | ✗     |
 | PAConv               | ✗        | ✗        | ✗        | ✓         | ✗     | ✗        | ✗     |
+| DGCNN               | ✗        | ✗        | ✗        | ☐         | ✗     | ✗        | ✗     |
```

Comment:
No need to add this line

Reply:
Actually DGCNN is also used as the backbone in some (though only a few) 3D segmentors/detectors. I think we should instead keep this line and add it as a potential backbone in this table? I mean changing the table header from

`|                    | ResNet   | ResNeXt  | SENet    |PointNet++ | HRNet | RegNetX | Res2Net |`

to

`|                    | ResNet   | ResNeXt  | SENet    |PointNet++ | DGCNN  | HRNet | RegNetX | Res2Net |`

After all, `PointNet++` is listed here